omigami/spectra_matching/spectra_matching.py

```diff
@@ -270,10 +273,6 @@ def _validate_input(self, model_input: List[Dict]):
 
     @staticmethod
     def _format_results(api_request: requests.Response) -> List[pd.DataFrame]:
-        def _sort_columns(df: pd.DataFrame):
-            sorted_columns = ["score"] + sorted(list(VALID_KEYS))
-            return df.reindex(columns=sorted_columns).dropna(axis=1, how="all")
-
```

Comment:
this is not valid anymore since we return all metadat by default right?

Reply:
yes